src/plugins/template/backend/abs.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "evaluates_map.hpp"
+#include <ngraph/runtime/reference/abs.hpp>
+
+using namespace ngraph;
+using namespace std;
+
+namespace {
+template <element::Type_t ET>
```

Comment:
The same of element::Type, let's use it from ov namespace. In future when we will remove legacy API, we won't have changes here

Reply:
Hi @ilyachur can we keep legacy API here? Let's make this PR just for split the file only. And than we can create another PR for removing legacy API. Because mixed the two things make the debug very difficult in the CI test error. How do u think?